locust-runner.py

```diff
@@ -1,34 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This script was never needed. I wrote it when I couldn't get env vars working in the yaml file but I upgraded docker-compose and switched some things (maybe it needed double quotes?); now it works.